src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/in/msg/ProcessedEnvelope.java

```diff
@@ -8,21 +8,17 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class ProcessedEnvelope {
 
-    public final UUID id;
-
     public final UUID envelopeId;
 
     public final Long ccdId;
 
     public final String envelopeCcdAction;
 
     public ProcessedEnvelope(
-        @JsonProperty("id") UUID id,
         @JsonProperty("envelope_id") UUID envelopeId,
         @JsonProperty("ccd_id") Long ccdId,
```

Comment:
not related to this change, but this should probably be a String

Reply:
Processor is sending Long. We should decide if we want String before this we should fix processor.

Reply:
you mean orchestrator?

Reply:
yes